plugins/node/instrumentation-amqplib/README.md

```diff
@@ -0,0 +1,99 @@
+| `publishHook`                  | `AmqplibPublishCustomAttributeFunction`    | hook for adding custom attributes before publish message is sent.                                             |
+| `publishConfirmHook`                  | `AmqplibPublishConfirmCustomAttributeFunction`    | hook for adding custom attributes after publish message is confirmed by the broker.                                             |
+| `consumeHook`                  | `AmqplibConsumerCustomAttributeFunction`    | hook for adding custom attributes before consumer message is processed.                                             |
+| `consumeEndHook`                  | `AmqplibConsumerEndCustomAttributeFunction`    | hook for adding custom attributes after consumer message is acked to server.                                             |
+| `consumeTimeoutMs`                  | `number`    | read [Consume Timeout](#ConsumeTimeout) below                                             |
+
+### Consume Timeout
+
+When user is setting up consume callback, it is user's responsibility to call ack/nack etc on the msg to resolve it in the server. If user is not calling the ack, the message will stay in the queue until channel is closed, or until server timeout expires (if configured).
+
+While we wait for the ack, a reference to the message is stored in plugin, which
+will never be garbage collected.
+To prevent memory leak, plugin has it's own configuration of timeout, which will close the span if user did not call ack after this timeout.
+
+If timeout is not big enough, span might be closed with 'InstrumentationTimeout', and then received valid ack from the user later which will not be instrumented.
+
+Default is 1 minute
+
+## Migration From opentelemetry-instrumentation-amqplib
+
+This instrumentation was originally published under the name `"opentelemetry-instrumentation-amqplib"` in [this repo](https://github.com/aspecto-io/opentelemetry-ext-js). Few breaking changes were made during porting to the contrib repo to align with conventions:
+
+### Hook Info
+
+The instrumentation's config `publishHook`, `publishConfirmHook`, `consumeHook` and `consumeEndHook` functions signature changed, so the second function parameter is info object, containing the relevant hook data.
+
+### `moduleVersionAttributeName` config option
+
+The `moduleVersionAttributeName` config option is removed. To add the amqplib package version to spans, use the `moduleVersion` attribute in hook info for `publishHook` and `consumeHook` functions.
+
+## Useful links
+
+- For more information on OpenTelemetry, visit: <https://opentelemetry.io/>
+- For more about OpenTelemetry JavaScript: <https://github.com/open-telemetry/opentelemetry-js>
+- For help or feedback on this project, join us in [GitHub Discussions][discussions-url]
+
+## License
+
+Apache 2.0 - See [LICENSE][license-url] for more information.
+
+[discussions-url]: https://github.com/open-telemetry/opentelemetry-js/discussions
```

Comment:
is it intended to point to core repo here?

Reply:
I think we don't have a discussion page for contrib, but use the core repo for both.
Let me know if I am missing anything.
Thanks

Reply:
Ah yes, you are right. Sorry.